source/_integrations/directv.markdown

```diff
@@ -48,3 +49,66 @@ Available services: turn_on, turn_off, media_play, media_pause, media_stop, medi
 | `entity_id`            |      yes | Target a specific media player. Defaults to all.                                                                                                                       |
 | `media_content_id`     |       no | The channel number to change to.                   |
 | `media_content_type`   |       no | A media type. Has to be `channel`.
+
+## Remote
+
+The DirecTV remote platform allows you to send remote control buttons to a DirecTV receiver. It is automatically setup when a DirecTV receiver is configured.
+
+At the moment, the following buttons are supported:
+
+- power
```

Comment:
I would suggest making those code blocks, as they are all values that can be used.

i.e.:
```suggestion
- `power`
```